resources/assets/js/components/AlertMessage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="alert ml-3" :class="{fade: !show, show: show, 'alert-success' : type=='success', 'alert-danger': type=='danger'}" role="alert" aria-labelledby="alertSuccess" :aria-hidden="!show">
+        <strong>{{ message }}</strong>
+        <!--<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>-->
+    </div>
+</template>
+
+<script>
+export default {
+  props: {
+    type: {
+      type: String,
+      required: false,
```

Comment:
Required en default need not both be present. The one implies the value of the other. Either set `required` to true, or have a default value